src/components/Feedback/Feedback.test.tsx

```diff
@@ -1,102 +1,32 @@
 import React from "react";
-import { fireEvent, render, screen } from "@testing-library/react";
+import { fireEvent, screen } from "@testing-library/react";
 import Feedback from "./Feedback";
-import userEvent from "@testing-library/user-event";
+import { feedbackRender } from "~/src/__tests__/testUtils/render";
 
 describe("Feedback", () => {
   beforeEach(() => {
-    render(<Feedback location={"/testing?testQuery"} />);
+    feedbackRender(<Feedback location={"/testing?testQuery"} />);
```

Comment:
Since it looks we only need `feedbackRender` in two places, my suggestion would be to simply wrap it in the `FeedbackProvider` directly in these spots, making the code a bit easier to read (no need to switch over to another file and see what `feedbackRender` does). Just a preference, but it is fine and works either way